### PR TITLE
Feature: Subjects Dashboard - Uncompleted

### PR DESCRIPTION
**Pull request by AngelZacarias**
Finishes #2 :
Cleans the frontend components and create basic navigation for the main dashboard. 
Deletes some dependencies in the frontend.

Starts #3 :
Starts the subjects dashboard functionallity including the basic models for graphql and the configuration for apollo. 

Still to do:
1. Filter the subjects for each student
2. Create a new subject manually
3. Open the subdashboard/submenu for each subject
4. Edit a subject

### CODE REVIEW

File: Client/src/routes.js [hunk 1]
```diff
@@ -1,113 +1,42 @@
...
-import RTLPage from "views/RTLPage/RTLPage.js";
+// Component Views
+import Subjects from "./views/General/Subjects";
+import About from "./views/General/About";
 
 const dashboardRoutes = [
   {
-    path: "/dashboard",
-    name: "Dashboard",
-    rtlName: "لوحة القيادة",
```
Comment (julioantonio12) [inline Client/src/routes.js:43]: Allahu akbar!

File: Server/graphql/typeDefs.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+module.exports = gql`
+    type Subject{
+        id: ID!
+        name : String!
+        color: String!
+        active: Boolean!
+        createdAt: String!
+    }
+    type Query{
+        getSubjects: [Subject]
```
Comment (julioantonio12) [inline Server/graphql/typeDefs.js:12]: Qué significa lo de [Subject] ? Significa que retorna un arreglo de Subjects?

File: Server/utils/check-auth.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+//const jwt = require('jsonwebtoken');
```
Comment (julioantonio12) [inline Server/utils/check-auth.js:1]: Al rato te digo una duda con este archivo

File: Client/src/routes.js (hunk 1 above)
Comment (AngelZacarias) [reply Client/src/routes.js:43]: Sera corregido en el futuro lol

File: Server/graphql/typeDefs.js (hunk 2 above)
Comment (AngelZacarias) [reply Server/graphql/typeDefs.js:12]: Si (: Aunque eso ya lo sabes

**Pull request merged**